locale/ar/client.json

```diff
@@ -3,6 +3,7 @@
   "action.change": "تغيير",
   "action.close": "إغلاق",
   "action.confirm": "تأكيد",
+  "action.copylink": "نسخ الرابط",
```

Comment:
Sorry, we don't manage changes to the localised strings here.

Please can you make the changes to the arabic strings here: https://crowdin.com/project/fider

Then they can be pulled into the next release

Thanks

Reply:
Sure I will upload an updated version, but I still have some modification on it along with some missing strings